Formula/spidermonkey.rb

```diff
@@ -1,19 +1,13 @@
 class Spidermonkey < Formula
   desc "JavaScript-C Engine"
   homepage "https://developer.mozilla.org/en/SpiderMonkey"
-  url "https://archive.mozilla.org/pub/mozilla.org/js/js185-1.0.0.tar.gz"
-  version "1.8.5"
-  sha256 "5d12f7e1f5b4a99436685d97b9b7b75f094d33580227aa998c406bbae6f2a687"
-  revision 1
+  url "https://people.mozilla.org/~sstangl/mozjs-38.2.1.rc0.tar.bz2"
+  version "38.2.1"
```

Comment:
Two PRs in the space of a year, the first being April 2015 apparently. Was reviewed within two days initially by Mike. Second was reviewed after a day by Alex. We try and wrap things quickly but sometimes it does take a while with back and forth comments if it's a confusing or problematic change.

If you leave a comment along the lines of "I'm busy with work, I'll pick this back up in a couple of weeks" it likely won't get closed, but otherwise we've no indication that you're still interested. Sadly, most people who don't respond for more than 10 days don't at all, over any length of time 😕.